app/templates/friends.html

```diff
@@ -2,31 +2,82 @@
 
 {% block content %}
 
-  <div class="friends-container">
+  <div class="friend-container">
+    <h1>Friends</h1> 
+    <button id="add-friend-btn" class="nav-btn">Add Friend</button>
 
-    <p>Friends</p>
+    <!---add friend ,odal window -->
+    <div id="add-friend-modal" class="modal">
+        <div class="modal-content">
+          <span class="close-btn">&times;</span>
 
-    <div class="friends-grid">
+          <!--Friend request form, edit to add route flask thing-->
+          <!--this is a placeholder and won't actually send a request -->
+          <!-- <form action="{{ url_for('friends') }}" method="POST" -->
+          <form onsubmit="alert('Friend request functionality not implemented yet'); return false;">
+            <input type="text" name="friend-username" placeholder="Enter friend's username" required>
+          </form>
 
-      {% for friend in friends %}
+          <!--wont really work, need a route for this, not sure if we will need a separate database for "friends" and "not friends"-->
+          <!--i added a temporary one in flask-->
+          <div class="friend-card"
```

Comment:
```suggestion
          <div class="friend-card">
```
Missing a closing bracket.